src/index.ts

```diff
@@ -49,3 +49,5 @@ export default plugin(({ theme, addVariant, prefix, e: escape }) => {
     });
   });
 });
+
+export default nestedGroupPlugin;
```

Comment:
Needed to make `ts-transform-default-export` work

https://github.com/axtgr/ts-transform-default-export/issues/1